test/uv-helper.mjs

```
@@ -0,0 +1,51 @@
+/**
+ * Shared UV helper for tests.
+ * UV is always installed into ~/.platformio/.cache (mirrors core.getCacheDir()).
+ */
+
+import { exec } from 'node:child_process';
+import { promisify } from 'node:util';
+import path from 'node:path';
+import os from 'node:os';
+import fs from 'node:fs';
```

ESLint

[error] 'fs' is defined but never used.

(no-unused-vars)